cnxdb/migrations/20170816204849_subcol-uuid-data.py

```diff
@@ -34,7 +34,8 @@ def up(cursor):
     if num_todo > 10:
         batch_size = int(num_todo / 10)
     else:
-        batch_size = num_todo
+        # Use a default of 1 to prevent a ValueError on range()
+        batch_size = num_todo or 1
 
     if batch_size > 10:
         batch_size = 10
```

Comment:
I was wondering, why don't we just set `batch_size` to 10?

Reply:
Works for me.

Reply:
Or are you suggesting we remove the math and only use a default of 10? Either way is fine by me.

Reply:
Yes I was thinking of removing the math.  Does it make sense without the math?

Reply:
Yeah! What are we actually gaining be for batches of say 7 vs 10? 👍